spec/active_record_with_lock_spec_helper.rb

```diff
@@ -28,3 +28,11 @@ def active_record_with_lock_adapter_connection
     lock_connection_pool: LockDatabaseRecord.connection_pool,
   )
 end
+
+RSpec.configure do |config|
+  if ENV["ADAPTER"] == "ActiveRecordWithLock"
+    config.filter_run_including :active_record_with_lock
+  else
+    config.filter_run_excluding :active_record_with_lock
+  end
+end
```

Comment:
I think it can be unintuitive that only some specs run. Given that this is to test adapter spec functionality, could we just add an RSpec for that class?